pkg/build/api/types.go

```diff
@@ -58,6 +58,11 @@ type BuildSpec struct {
 
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
+
+	// Optional duration in seconds, counted from the time when a build pod gets
```

Comment:
if this is optional, maybe it would be worth to explain what is the default behavior if I don't set this